### ISSUE TITLE
Add Quota Support

### ISSUE THREAD

**Issue by smashingraph** [myoung34/vagrant-ovirt3#38]
Would it be possible to add quota support ?
REST API allows to pass a quota UUID:
```
<parameter required="false" type="xs:string">
<name>vm.quota.id</name>
</parameter>
``` 

This would allow to create a VM in a Quota

**Comment by myoung34** [comment]
It shouldn't be an issue, ``rbovirt`` supports quota usage, I'll just have to find time to sit down and look

**Comment by myoung34** [comment]
Resolved in version `1.6.0`
